README.md

```diff
@@ -26,11 +29,12 @@ When a new version of CosmosSDK is published, we may want to adopt the changes i
 
 1. Ensure the `main` branch and all tags are up to date by following the steps above in "Fork maintenance".
 2. Create a new branch off the desired CosmosSDK commit using tags. `git checkout -b dydx-fork-$VERSION <CosmosSDK repo's tag name>`. The new branch should be named something like `dydx-fork-$VERSION` where `$VERSION` is the version of CosmosSDK being forked (should match the CosmosSDK repo's tag name). i.e. `dydx-fork-v0.47.0-alpha2`.
-3. Push the new branch.
-4. Open a PR which cherry-picks each commit in the current default branch, in order, on to the new `dydx-fork-$VERSION` branch (note: you may want to consider creating multiple PRs for this process if there are difficulties or merge conflicts). For example, `git cherry-pick <commit hash>`.
-5. Get approval, and merge.
-6. Update `dydxprotocol/v4` by following the steps in "Making Changes to the fork" above.
-7. Set `dydx-fork-$VERSION` as the [default branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch) in this repository.
+3. Push the new branch (i.e `dydx-fork-v0.47.0-alpha2`).
+4. Off of this new branch, create a new branch. (i.e `totoro/dydxCommits`)
+5. Cherry-pick each dydx-created commit from the current default branch, in order, on to the new `dydx-fork-$VERSION` branch (note: you may want to consider creating multiple PRs for this process if there are difficulties or merge conflicts). For example, `git cherry-pick <commit hash>`. You can verify the first commit by seeing the most recent commit sha for the `$VERSION` (i.e `v0.47.0-alpha2`) tag on the `cosmos/cosmos-sdk` repo, and taking the next commit from that sha.
+6. Open a PR to merge the second branch (`totoro/dydxCommits`) into the first (`dydx-fork-v0.47.0-alpha2`). Get approval, and merge.
+7. Update `dydxprotocol/v4` by following the steps in "Making Changes to the fork" above.
+8. Set `dydx-fork-$VERSION` as the [default branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch) in this repository.
```

Comment:
Should we add a note about the GitHub workflow permissioning and an example of those errors? It requires a GitHub admin to whitelist any new workflows.

Reply:
Done